cpg_workflows/large_cohort/site_only_vcf.py

```diff
@@ -8,18 +8,171 @@
 import hail as hl
 
 from cpg_utils import Path, to_path
-from cpg_utils.config import config_retrieve
+from cpg_utils.config import config_retrieve, dataset_path, output_path
 from cpg_workflows.batch import override_jar_spec
 from cpg_workflows.utils import can_reuse
 from gnomad.utils.sparse_mt import default_compute_info
 from gnomad.utils.vcf import adjust_vcf_incompatible_types
 
 
+def extract_as_pls(
+    lpl_expr: hl.expr.ArrayExpression,
+    allele_idx: hl.expr.Int32Expression,
+) -> hl.expr.ArrayExpression:
+    """
+    Function pulled from `gnomad_qc/v4/annotations/generate_variant_qc_annotations.py`:
+        - CPG's `gnomad_methods` submodule is pinned to a commit that pre-dates this function
+        and thus cannot import it directly.
```

Comment:
Noting that the `gnomad_methods` and `gnomad_qc` are different resources
```suggestion
    Function pulled from `gnomad_qc/v4/annotations/generate_variant_qc_annotations.py`.
```